src/main/webapp/js/instructorFeedbackResultsAjaxByQuestion.js

```diff
@@ -25,22 +25,23 @@ $(document).ready(function() {
             },
             success: function(data) {
                 var appendedQuestion = $(data).find('#questionBody-0').html();
+                var $panelBody = $(panelBody[0]);
```

Comment:
Remove the '$' in front of the variable name? It does not follow the naming convention, and is confusing (with the jquery function $).